src/ConcertDisplay.js

```diff
@@ -6,15 +6,29 @@ export default class ConcertDisplay extends Component {
   constructor() {
     super();
   }
-
+  
   render() {
-    const {concertData, setlistData} = this.props;
-    return (
-      <main className="concert-display">
-      {concertData.map(concert => {
-        return <Concert concert={concert}/>
-      })}
-      </main>
-    )
+    const {concertData, setlistData, currentSearch} = this.props;
+    if(currentSearch === null) {
+      return (
+        <main className="concert-display">
+        {this.props.concertData.map(concert => {
+          return <Concert concert={concert}/>
+          })}
+        </main>
+      )
+    } else {
```

Comment:
We'll have to think about how to add logic so that the display will not change if we're in the expanded view... Maybe adding to the App state to say what view we're in and having the search results display be dependent upon that.

Reply:
Or we could make the card collapse, if it is expanded when the user types into the search box. I'm thinking if they want to search for something, then they are done looking at the expanded card, and aren't going to want it expanded anyways.